content/en/monitors/notify/variables.md

```diff
@@ -315,6 +315,7 @@ To include **any** attribute or tag from a log, a trace span, a RUM event, a CI
 |-----------------|--------------------------------------------------|
 | Log             | `{{log.attributes.key}}` or `{{log.tags.key}}`   |
 | Trace Analytics | `{{span.attributes.key}}` or `{{span.tags.key}}` |
+| Error Tracking  | `{span.attributes.[error.message]}}`             |
```

Comment:
I could be wrong, but are we missing an { at the beginning of the value?

Reply:
```suggestion
| Error Tracking  | `{{span.attributes.[error.message]}}`             |
```